tests/acceptance/features/apiSearchContent/extractedProps.feature

```diff
@@ -255,3 +255,139 @@ Feature: propfind extracted props
           }
         }
       """
+
+
+  Scenario: GET extracted properties of an audio file (Project space)
+    Given using spaces DAV path
```

Comment:
```suggestion
    Given using spaces DAV path
```

This has already been mentioned in the `Background`. So I think we can remove this from here.